src/main/java/mate/jdbc/util/ConnectionUtil.java

```diff
@@ -0,0 +1,27 @@
+package mate.jdbc.util;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+    static {
+        try {
+            Class.forName("com.mysql.cj.jdbc.Driver");
+        } catch (ClassNotFoundException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    public static Connection getConnection() {
+        try {
+            Properties dbProperties = new Properties();
+            dbProperties.put("user", "root");
```

Comment:
It's not recommended to define properties of db, like username or password directly in Java code, because it's sensitive information, I would recommend adding some properties files and secure them from inappropriate access. If it's not clear for you, we can discuss it.